packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -667,8 +667,18 @@ abstract class CreateBase extends FlutterCommand {
       'templates',
       'template_manifest.json',
     );
+    late final String manifestFileContents;
```

Comment:
could this have been `final`? I know it's initialized in a `try` block, but the compiler should know that `throwToolExit` should `Never` return.

Reply:
Indeed! Good catch.

Reply:
Addressed in [avoid late final declaration when final is sufficient](https://github.com/flutter/flutter/pull/145620/commits/d613ea097f8d8db80bf716e03ada1a39a457af26)